tests.edn

```diff
@@ -0,0 +1,11 @@
+#kaocha/v1
+ {:reporter [kaocha.report/documentation]
```

Comment:
For a small project like this one you can go directly to the empty map I think

Reply:
I simplified, but the doc output is quite nice, especially when stuff breaks